src/bundle/Resources/public/scss/_popup-menu.scss

```diff
@@ -5,6 +5,12 @@
     border-radius: $ibexa-border-radius;
     box-shadow: calculateRem(4px) calculateRem(22px) calculateRem(67px) 0 rgba($ibexa-color-info, 0.2);
```

Comment:
Maybe we'll add this style to the variable?

Reply:
Currently this shadow is used only here, so I think it should not be extracted as a global variable.